pnpm-workspace.yaml

```diff
@@ -1,3 +1,17 @@
 packages:
   - 'packages/*'
   - 'sites/*'
+
+catalog:
+  # Frameworks
+  '@zag-js/svelte': ^0.65.0
+  '@zag-js/react': ^0.65.0
+
+  # Machines
+  '@zag-js/accordion': ^0.65.0
+  '@zag-js/avatar': ^0.65.0
+  '@zag-js/rating-group': ^0.65.0
+  '@zag-js/tabs': ^0.65.0
+  '@zag-js/progress': ^0.65.0
+  '@zag-js/radio-group': ^0.65.0
+  '@zag-js/switch': ^0.65.0
```

Comment:
we may also want to add a `typescript` to the catalog too given that it's used in every package

Reply:
I was wondering this too, Chris agreed in the discord channel so I'll go ahead and add it too. Should we do latest stable? `5.5.4`

Reply:
yea, updating to the latest should be fine

Reply:
Done.

Reply:
@Hugos68 when you feel this is complete and ready for review, go ahead and mark "ready for review" and add Adrian as a reviewer please.